src/realm/util/file_mapper.cpp

```diff
@@ -368,12 +485,15 @@ void msync(void* addr, size_t size)
     // See also
     // https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/CoreData/Articles/cdPersistentStores.html
     // for a discussion of this related to core data.
+
+#ifdef _WIN32
+    
```

Comment:
Program execution could not hit that point on Windows. I've just cleaned that up in https://github.com/realm/realm-core/pull/2433/commits/d2db5b8b5935354bd4a7b2c8f2c462038baafbef